cmd/crypto/retry.go

```diff
@@ -21,8 +21,8 @@ import (
 
 // default retry configuration
 const (
-	retryWaitMin = 500 * time.Millisecond // minimum retry limit.
-	retryWaitMax = 3 * time.Second        // 3 secs worth of max retry.
+	retryWaitMin = 500 * time.Millisecond  // minimum retry limit.
```

Comment:
Maybe reduce this to 100ms. Starting with 500ms is quite a lot.

Reply:
Yeah 100ms would work too @aead

Reply:
Sure, reduced the min to `100ms`